### ISSUE TITLE
[OFFICIAL] Kubescape Headlamp plugin

### ISSUE THREAD

**Issue by mgalesloot** [artifacthub/hub#4046]
Pleaes provide the official status to the Kubescape Headlamp plugin. 

- **Repository name** *(in `artifacthub.io`)*: kubescape-headlamp-plugin (ID: 17c5b888-af3c-4279-a4cd-c8e39f8d0be3) 
- **Official packages** *(leave empty if all packages in the repository are official)*: 
- **Project URL:** https://github.com/kubescape/headlamp-plugin
- **Is the publisher a CNCF project? (graduated, incubating or sandbox):** yes 
- **Source code URL** *(if available)*: https://github.com/kubescape/headlamp-plugin
- **Other relevant URLs:** 

**Before applying for this status, please make sure your repository complies with the following requirements:**

- [x] The repository has already obtained the [Verified Publisher](https://artifacthub.io/docs/topics/repositories/#verified-publisher) status.
- [x] The user requesting the status is the publisher of the repository in Artifact Hub, or belongs to the organization publishing it.
- [x] All official packages available in the repository provide a `README.md` file with some documentation that can be displayed on Artifact Hub.

**Comment by tegioz** [comment]
Hi @mgalesloot 👋

I'm not sure if the official status applies to this case. For this plugin to be marked as official, it should be owned and published by the Kubescape project, as they own the software this package primarily focuses on.

Hope this makes sense 🙂

**Comment by mgalesloot** [comment]
Yes agreed, this make sense. Closing this request for now.

**Comment by mgalesloot** [comment]
We have moved the code of the Kubescape Headlamp plugin to https://github.com/kubescape/headlamp-plugin. The code is now part of the Kubescape project. Please reconsider the request to grant the official status to the Kubescape Headlamp plugin.

**Comment by matthyx** [comment]
Indeed, Matthijs is now an official Kubescape maintainer and we've moved the repo to our org.
Thanks folks !

**Comment by tegioz** [comment]
Awesome, thanks!

Could you setup the Kubescape organization in Artifact Hub and transfer the repository to it?

**Comment by mgalesloot** [comment]
I have created the Kubescape organization ikn artifacthub, but fail to add the Kubescape Headlamp plugin to it. For some reason the repository cannot be selected, it is grayed out. I can select any other repo, but not kubescape-plugin. 

![Image](https://github.com/user-attachments/assets/2b4c248c-938b-4614-8466-3c2e5aa084a8)

**Comment by cynthia-sg** [comment]
You can transfer your repo from your account to the organization's account.

<img width="216" alt="Image" src="https://github.com/user-attachments/assets/6a2a83ca-f5f3-42ee-bba8-6b34cdb75e1a" />

**Comment by mgalesloot** [comment]
The repository is now part of Kubescape organization: 
https://artifacthub.io/packages/headlamp/kubescape-headlamp-plugin/headlamp_kubescape

**Comment by tegioz** [comment]
Thank you! Marked as official -and CNCF- 🙂
